Dockerfile

```diff
@@ -20,7 +28,7 @@ ENV STREAMLIT_CLIENT_SHOW_ERROR_DETAILS=false \
     STREAMLIT_BROWSER_GATHER_USAGE_STATS=false
 
 COPY ./demo_app/requirements.txt .
-RUN pip install --progress-bar off --no-cache-dir -r requirements.txt && \
+RUN uv pip install --no-progress --no-cache -r requirements.txt && \
```

Comment:
Switching from pip to uv changed duration for this specific line from [24s](https://app.circleci.com/pipelines/github/civisanalytics/civis-services-streamlit/54/workflows/05ab9e6a-1a33-4cbe-ba37-1c1289e3d6d2/jobs/51) to [3s](https://app.circleci.com/pipelines/github/civisanalytics/civis-services-streamlit/59/workflows/42aee8d1-334f-4315-a976-b790b607323c/jobs/56). (The [first commit](https://github.com/civisanalytics/civis-services-streamlit/pull/4/commits/008e00b89f95d2a80b2302e5653e9fe9232dad50) in this PR is intentionally an empty commit to just trigger the CI build for comparison.)